docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.7"
+
+services:
+  marklogic:
+    image: marklogiccommunity/marklogic:9.0-8
+    volumes:
+      - type: volume
+        source: marklogic
+        target: /data
+    ports:
+      - "8000-8002:8000-8002/tcp"
+      - "8095:8095/tcp"
+    expose:
+      - "7997"
+      - "7998"
+      - "7999"
+
+  feature-server:
+    image: marklogiccommunity/feature-server:1.0.0-beta.4
```

Comment:
Have published this as a private repository.  Once you have kicked the tires I can make it public.